frontend/src/views/ParticipantList.js

```diff
@@ -65,9 +81,55 @@ const ParticipantList = observer(() => {
         </Link>
         <PrevPointCopy>Search Results</PrevPointCopy>
       </Breadcrumbs>
-      <PrevPointHeading className={classes.participantsListHeading}>
-        Participants
-      </PrevPointHeading>
+
+      <Grid
+        container
+        spacing={1}
+        alignItems="flex-end"
+        className={classes.autocompleteWrapper}
+      >
+        <Grid item>
+          <PrevPointHeading className={classes.participantsListHeading}>
+            Participants
+          </PrevPointHeading>
+        </Grid>
+
+        <Grid item>
+          {participantStore.participants.length > 0 && (
```

Comment:
just `participantStore.participants` is good enough here. if the value is truthy it will display.